pytest_rerunfailures.py

```diff
@@ -513,23 +512,29 @@ def pytest_runtest_teardown(item, nextitem):
         # flaky
         return
 
+    if hasattr(item, "execution_count") is False:
```

Comment:
```suggestion
    if not hasattr(item, "execution_count"):
```

This should be enough I see no need to check for identity with `False`.